lib/fpm/util.rb

```diff
@@ -226,26 +232,90 @@ def safesystemout(*args)
     return stdout_r_str
   end # def safesystemout
 
+  # Get an array containing the recommended 'ar' command for this platform
+  # and the recommended options to quickly create/append to an archive
+  # without timestamps or uids (if possible).
+  def ar_cmd
+    return @@ar_cmd if defined? @@ar_cmd
+
+    @@ar_cmd_deterministic = FALSE
```

Comment:
TIL Ruby provides a constant `FALSE` that means `false`. This should probably be lowercase `false` (for style) but there is no negative impact to the user, so it's cool.